cranelift/codegen/src/ir/pcc.rs

```diff
@@ -128,36 +128,52 @@ pub enum PccError {
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
 #[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
 pub enum Fact {
-    /// A bitslice of a value (up to a bitwidth) is less than or equal
-    /// to a given maximum value.
+    /// A bitslice of a value (up to a bitwidth) is within the given
+    /// integer range.
     ///
     /// The slicing behavior is needed because this fact can describe
     /// both an SSA `Value`, whose entire value is well-defined, and a
     /// `VReg` in VCode, whose bits beyond the type stored in that
     /// register are don't-care (undefined).
-    ValueMax {
+    ValueRange {
```

Comment:
:bike: :paintbrush:  Maybe we should just call this `Range`? That matches the text format.

Reply:
Ah, yeah, that makes more sense!